BeagleBone/AI/pru/inputR31.c

```diff
@@ -0,0 +1,35 @@
+// Reads input in P8_13 via the R31 register and blinks the USR3 LED
```

Comment:
A ring oscillator implementation is also interesting. 

* https://github.com/beagleboard/cloud9-examples/blob/master/extras/pru/ring.pru0c
* https://pub.pages.cba.mit.edu/ring/